web/concrete/blocks/google_map/controller.php

```diff
@@ -1,76 +1,89 @@
-<?
+<?php
 namespace Concrete\Block\GoogleMap;
+
 use Loader;
 use Page;
 use \Concrete\Core\Block\BlockController;
-class Controller extends BlockController {
-	
-	protected $btTable = 'btGoogleMap';
```

Comment:
that's what the multiple commits are for... check out https://github.com/ExchangeCore/concrete5-5.7.0/commit/ef20524c2cc54e6f6b74413fcf6d7f7728e94818 that has the real changes in it